### ISSUE TITLE
Replace MultipleSelect with component from design system

### ISSUE THREAD

**Issue by lorang92** [Altinn/app-frontend-react#510]
### Description

In https://github.com/Altinn/app-frontend-react/pull/505 we implemented a multiple select component in app-frontend. This component was implemented directly as a result of time pressure. Once this component is [implemented in design system](https://github.com/Altinn/altinn-design-system/issues/138), this implementation should be replaced.

### Tasks

- [ ] Replace existing implementation with new
- [ ] The current multiple select only supports storing data as a comma-separated string. A useful feature here would be to store this as an array of selected values, reusing the likert-component implementation with "list" binding. Consider adding this.
- [ ] Update user docs if relevant

### Acceptance Criterias

- Custom implementation is replaced with component from design system

**Comment by olemartinorg** [comment]
Unblocked by #831

**Comment by bjosttveit** [comment]
The new release introduced a new bug in the single select variant that causes the "default" value not to show up even though it is set correctly. This impacts the language selector and causes it to be blank on refresh.

![Image](https://user-images.githubusercontent.com/47412359/227927812-16a22e77-c087-4fe5-a252-c3f695f3a2ce.png)

**Comment by bjosttveit** [comment]
Blocked by: https://github.com/digdir/designsystem/pull/286
